src/wirecloud/defaulttheme/templates/wirecloud/workspace/widget.html

```diff
@@ -2,6 +2,7 @@
 <div class="fade panel panel-default">
     <div class="wc-widget-heading panel-heading"><h4 class="panel-title"><t:title/></h4><div class="wc-widget-infobuttons"><t:errorbutton/></div><div class="wc-widget-buttons"><t:minimizebutton/><t:menubutton/><t:closebutton/></div></div>
     <div class="wc-widget-body"><t:iframe/></div>
+    <t:leftsideresizehandle/><t:rightsideresizehandle/>
```

Comment:
I put them outside the body because the body has `overflow: hidden` so the 6px outside would not work. This seems to work pretty well